wcivf/apps/elections/migrations/0025_auto_20190412_1528.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def insert_voting_systems(apps, schema_editor):
+    VotingSystem = apps.get_model("elections", "VotingSystem")
+    db_alias = schema_editor.connection.alias
+    VotingSystem.objects.using(db_alias).bulk_create(
+        [
+            VotingSystem(
```

Comment:
This is a straight copy & paste job from
https://github.com/DemocracyClub/EveryElection/blob/cfe79ec3b8d5b9da057ff3f3b84f9ff5841e0749/every_election/apps/elections/constants.py
It would probably be nicer if we could get this from an API endpoint or something, but for now we can't.